lib/streamlit/watcher/event_based_path_watcher.py

```diff
@@ -12,31 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Declares the EventBasedFileWatcher class, that watches the file system.
+"""Declares the EventBasedPathWatcher class, which watches given paths in the file system.
 
 How these classes work together
 -------------------------------
 
-- EventBasedFileWatcher : each instance of this is able to watch a single
-  files so long as there's a browser interested in it. This uses
-  _MultiFileWatcher to watch files.
+- EventBasedPathWatcher : each instance of this is able to watch a single
+  file or directory at a given path so long as there's a browser interested in
+  it. This uses _MultiPathWatcher to watch files.
 
-- _MultiFileWatcher : singleton that watches multiple files. It does this by
+- _MultiPathWatcher : singleton that watches multiple paths. It does this by
```

Comment:
I'm confused how this watches files. In line 22, it says this class watches files but doesn't seem to say so.

Reply:
I just missed a substitution from files->paths

Will fix in a later commit because I don't think fixing it here is worth the hassle of rebasing